scripts/approvalYears.py

```diff
@@ -998,15 +1001,15 @@ def writeCBERBLAs(fpout, uniiPT, uniiALL):
     header = 'NDA\tProduct\tForm;Route\tStrength\tStatus\tAppl Type\tSponsor\tDate\tDate Ref\tUNIIs\tIngredients\n'
     fp.write(header)
     
-    for prod in prods.keys():
+    for prod in list(prods.keys()):
```

Comment:
Same as above.

Reply:
Python3 versus python2 issue. 2to3 automagically has made this change as .keys() no longer returns a list in python3